### ISSUE TITLE
Visualisation of scATAC-seq's Peak

### ISSUE THREAD

**Issue by Starlitnightly** [huidongchen/simba#11]
Dear Huidong,

I hope this letter finds you well. I am writing to seek your guidance on visualizing specific chromosome regions in Python for scATAC-seq analysis using SIMBA.

I have been utilizing SIMBA in my research on single-cell genomics, and I am impressed by the comprehensive framework you have developed. However, I have encountered a challenge in visualizing chromosome regions of interest, as this aspect is not extensively described in the publicly available code.

Given your expertise in the field and your deep understanding of SIMBA, I would greatly appreciate any advice or recommendations you could provide on visualizing specific chromosome regions within the SIMBA framework. Any code snippets or references to relevant parts of the codebase would be immensely helpful.

Thank you for your time and contribution to the scientific community. I am grateful for the opportunity to learn from your expertise and look forward to your response.

Sincerely,

Zehua

<img width="438" alt="image" src="https://github.com/huidongchen/simba/assets/46667721/6148d1a4-37d8-4f2a-8b70-cb3965a329ce">

**Comment by huidongchen** [comment]
Hi Zehua,

Thank you for your interest in SIMBA and for your kind words! Sorry for the delay.

Unfortunately the visualization of peaks was not done through an automated process. 

Instead, what I did was simply utilize [IGV](https://software.broadinstitute.org/software/igv/) to display the reads associated with each cell type within the specific region of interest. Afterwards, I saved the output from IGV. 

While this may not be what you were looking for, I hope it still provides some assistance.
